pages/attributes.vue

```diff
@@ -0,0 +1,342 @@
+      { 
+        return std::accumulate(value.begin(), value.end(), 0);
+      } 
+    };
+    </code>
+    <!-- eslint-enable -->
+
+    <p
+      :class="paragraph"
+    >
+    or
+    </p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+    template <> 
+    struct GenericAttributeConversion< MyType > 
+    { 
+      static float converted_value( const MyType& value ) 
+      { 
+        return value.get_a_float(); 
+      } 
+    };
+    </code>
+    <!-- eslint-enable -->
+
+    <h2 :class="section">Modify attribute size</h2>
+
+    <p
+      :class="paragraph"
+    >
+    To modify the attribute size, you should use the <span v-html="attributeManager" />.
+    The first solution is to use the method <code>resize</code> as shown above.
+    The second solution is to select the element to remove:
+    </p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+      AttributeManager manager;
+      manager.resize( 6 );
+      auto attribute_ptr = manager.find_or_create< VariableAttribute, double >(
+        "first_attribute", 12.3 );
+      std::vector< bool > to_delete( 6, false );
+      to_delete[0] = true;
+      to_delete[3] = true;
+    </code>
+    <!-- eslint-enable -->
+
+    <p
+      :class="paragraph"
+    >
+    Here the elements 0 and 3 are removed from all the 
+    attributes associated to the <span v-html="attributeManager" />.
+    </p>
+
+    <h2 :class="section">Modify attribute values</h2>
+
+    <p
+      :class="paragraph"
+    >
+    To modify the value of attribute elements, 
+    you may use either the attribute itself or 
+    its associated <span v-html="attributeManager" />. 
+    In the code below, we present four ways to modify attribute values:
+    </p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+      AttributeManager manager;
+      manager.resize( 6 );
+      auto attribute_ptr = manager.find_or_create< VariableAttribute, double >(
+        "first_attribute", 12.3 );
+      // values: 12.3 12.3 12.3 12.3 12.3 12.3 
+      
+      attribute_ptr->set_value( 1, 5.0 );
+      // values: 12.3 5.0 12.3 12.3 12.3 12.3 
+
+      const auto factor = 2;
+      attribute_ptr->modify_value( 4, [&factor]( double& value ){ value *= factor; } );
+      // values: 12.3 5.0 12.3 12.3 24.6 12.3 
+
+      manager.assign_attribute_value( 1, 2 );
+      // values: 12.3 5.0 5.0 12.3 24.6 12.3 
+
+      const AttributeLinearInterpolation interpolator{ {2, 4}, {0.5, 0.5} };
+      manager.interpolate_attribute_value( 0, interpolator );
+      // values: 14.8 5.0 5.0 12.3 24.6 12.3 
+    </code>
+    <!-- eslint-enable -->
+
+    <p
+      :class="paragraph"
+    >
+    If you use <span v-html="attributeManager" /> methods, 
+    the modification will be operated on all associated attributes.
+    </p>
+
+  </v-layout>
+</template>
+
+<script>
+export default {
+  data: () => ({
+    section: 'display-2 ma-5 pa-5',
```

Comment:
Not related to this PR but we should define a mixin to have the common data stored inside the same component.